examples/regexp.fx

```diff
@@ -0,0 +1,104 @@
+import Re2
+
+fun example(funname: string, arg1: 'a, arg2: 'b, f: ('a, 'b) -> 't): void
+{
+	println(f"Re2.{funname}(\"{arg1}\", \"{arg2}\") = {f(arg1, arg2)}")
+}
+
+//fun bugreport1(arg1: string, arg2: Re2.regex_t, f: (string, Re2.regex_t) -> 't): void
+//{
+//	println(f"{f(arg1,arg2)}")	
+//}
+//val reg_br = Re2.compile("h.*o")
+//bugreport1("hello", reg_br, Re2.full_match)
```

Comment:
Since `Re2.full_match` has several overloaded variants, a specific instance needs to be specified, e.g.
```
bugreport1("hello", reg_br, (Re2.full_match: (string, Re2.regex_t)->bool))
```

Reply:
Ok.